packages/squiggle-lang/package.json

```diff
@@ -4,6 +4,7 @@
   "homepage": "https://squiggle-language.com",
   "license": "MIT",
   "scripts": {
+    "peggy": "peggy --cache ",
```

Comment:
No because it is intentionally a command that can operate on different parse files as an argument. Yarn entry is to make the command available.
If we want to have a command that processes all parse files in the project then we need a script. Should I write one?